README.rst

```diff
@@ -323,6 +323,55 @@ https://github.com/onetrueawk/awk is a severly limited version awk that some pri
    RUN apt-get update; apt-get install vim
    COPY --from=onetrueawk /usr/local /usr/local
 
+GDAL
+----
+
+============ ============
+Name         GDAL
+Build Args   ``GDAL_VERSION`` - Version of GDAL to download
+Output files ``/gdal/usr/local/*``
+============ ============
+
+Compiles GDAL v3, including PROJ v6, ECW J2K 5.5, OPENJPEG 2.3
+
+.. rubric:: Example
+
+.. code-block:: Dockerfile
+
+   FROM python:3.6.9-slim-jessie as python
+   FROM vsiri/recipe:gdal as gdal
+   FROM ubuntu:16.04
+
+   # set shell to bash
+   SHELL ["/usr/bin/env", "/bin/bash", "-euxvc"]
+
+   # install python & gdal
+   COPY --from=python /usr/local /usr/local/
+   COPY --from=gdal /gdal/usr/local /usr/local
+   RUN ldconfig
```

Comment:
I know it seems silly, but should this be put into a ["patch"](https://github.com/VisionSystemsInc/docker_recipes/blob/master/recipe_pipenv.Dockerfile#L8)? I'm doing `git lfs install` for git-lfs, a similar one line patch

Reply:
I can add that for those who want it, but maybe the example shouldn't require the VSI recipe too?